persistence/src/commonMain/kotlin/com/wire/kalium/persistence/dao/call/CallDAO.kt

```diff
@@ -0,0 +1,35 @@
+package com.wire.kalium.persistence.dao.call
+
+import com.wire.kalium.persistence.dao.ConversationEntity
+import com.wire.kalium.persistence.dao.QualifiedIDEntity
+import kotlinx.coroutines.flow.Flow
+
+data class CallEntity(
+    val conversationId: QualifiedIDEntity,
+    val id: String,
+    val status: Status,
+    val callerId: String,
+    val conversationType: ConversationEntity.Type
+) {
+    enum class Status {
+        STARTED,
+        INCOMING,
+        MISSED,
+        ANSWERED,
+        ESTABLISHED,
+        STILL_ONGOING,
+        CLOSED
+    }
+}
+
+interface CallDAO {
+    suspend fun insertCall(call: CallEntity)
+    suspend fun getCalls(): Flow<List<CallEntity>>
+    suspend fun getIncomingCalls(): Flow<List<CallEntity>>
+    suspend fun getEstablishedCalls(): Flow<List<CallEntity>>
+    suspend fun getOngoingCalls(): Flow<List<CallEntity>>
```

Comment:
I think this, accordingly to what @trOnk12 was talking the other day, should be called with prefix `observeXXXX` 🤔

Reply:
Great catch!